Makefile

```diff
@@ -17,9 +16,12 @@ endif
 .PHONY: help
 help:  ## This.
 	@perl -ne 'print if /^[a-zA-Z_-]+:.*## .*$$/' $(MAKEFILE_LIST) \
-	| sort \
```

Comment:
By not sorting the help text, the targets will be listed in the order they appear.
This allows us to tailor the output to our liking.

Reply:
nice!